src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-rtree.py

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
```

Comment:
Final nitpick: it's 2021 (even though these days time might appear to have frozen).

Reply:
holy i copied the wrong one